lang/resources/org/partiql/type-domains/partiql.ion

```diff
@@ -453,7 +453,7 @@ may then be further optimized by selecting better implementations of each operat
         (sum type
 
             // Scalar type
-            (scalar_type id::symbol parameters::(* int 0))
+            (scalar_type alias::symbol parameters::(* int 0))
```

Comment:
This is now considered a breaking change as well

Reply:
Yeah, thanks for pointing that out. This PR is targeting `PSTS-FB` instead of `main`, and my plan is to document all the breaking changes once we merge `PSTS-FB` to `main`. Since `PSTS-FB` is still kind of an experimental branch for now, it might not worth the effort to document everything in PRs targeting `PSTS-FB`. (we may want to modify or even rollback these breaking changes later)